source/aks/upgrading-aks-clusters.html.md.erb

```diff
@@ -23,12 +23,32 @@ This is the normal order we use when upgrading clusters, it does not have to exa
 - Production       - (prod-00-aks, prod-01-aks/ ss-prod-00-aks, ss-prod-01-aks) 
 - Ptl              - (cft-ptl-00-aks/ ss-ptl-00-aks) 
 
-## Pods and HRs Health Checks
+## Planning
+
+Before starting to upgrade any clusters, we need to find out if the version of kubernetes we are upgrading to has any deprecations in it that might affect our applications.
+
+We can do this using the  [pluto tool](https://github.com/FairwindsOps/pluto).
+
+Run pluto against the prod and ptl clusters to ensure there are no resources using apiVersions that will be deprecated in the kubernetes version you are upgrading to.
+
+```command
```

Comment:
Worth keeping the point about being connected to a cluster / the correct cluster before running this command?

Reply:
Makes sense. Added it twice just to reiterate in the different steps